microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md

```diff
@@ -48,39 +46,26 @@ The labs are also designed to be connected to trials for:
 
 ## Step-by-step labs
 
-Detailed lab guides take you through multiple deployment and management scenarios. The labs have been updated for the latest versions of Intune and Configuration Manager.
-
-Note: The Windows 11 version of the lab includes the Windows 10 lab guide. A Windows 11 version of the lab guide will be available soon. 
+Detailed lab guides take you through multiple deployment and management scenarios. The labs have been updated for the latest versions of Intune and Configuration Manager. Note: A new Windows 11 version of the lab is now available. The lab guides include the following scenarios: 
 
 ### Plan and prepare infrastructure
 
 - Desktop Analytics
-- Cloud Management Gateway & Cloud Distribution Point
-- NEW! Tenant attach and co-management
-- NEW! Endpoint analytics
-- Remote access (VPN)
-
-### Prepare configuration
-
-- Optimize Windows 10 update delivery
-- Servicing Windows 10 using Group Policy
-- Servicing Windows 10 using Microsoft Intune
-- Servicing Windows 10 with Configuration Manager
-- Servicing Microsoft 365 Apps for enterprise using Configuration Manager
-- Servicing Microsoft 365 Apps for enterprise using Intune
-- Security and compliance
+- Cloud Management Gateway 
+- Tenant attach and co-management
+- Endpoint analytics
+- Optimize update delivery
 
-### Prepare applications
+### Deploy Windows 
 
-- Readiness Toolkit for Office
-- MSIX Packaging and Conversion of Win32 applications
+- OS deployment task sequences in Configuration Manager
+- Windows Autopilot
 
-### Deploy Windows 10
+### Service Windows 
 
-- OS Deployment task sequences in Configuration Manager
-- OS Deployment task sequences in the Microsoft Deployment Toolkit (MDT)
-- Windows Autopilot
-- Deploy and manage the new Microsoft Edge
+- Servicing Windows using Group Policy
+- Servicing Windows using Microsoft Intune
+- Servicing Windows 10 with Configuration Manager
```

Comment:
Should this just be "Servicing Windows with Configuration Manager" and not version specific?